miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -411,6 +407,40 @@ open class MiniAppMessageBridge {
         bridgeExecutor.postError(callbackId, "${ErrorBridgeMessage.ERR_SHARE_CONTENT} ${e.message}")
     }
 
+    @VisibleForTesting
+    internal fun onSendJsonToHostApp(callbackObj: CallbackObj)  {
+        sendJsonToHostApp(
+            content = callbackObj.param,
+            onSuccess = { value ->
+                bridgeExecutor.postValue(callbackObj.id, value.toString())
+            },
+            onError = { message ->
+                bridgeExecutor.postError(
+                    callbackObj.id,
+                    message
+                )
+            }
+        )
+    }
+
+    /**
+     * handle Universal Bridge that represented as a json  from MiniApp.
+     **/
+    open fun sendJsonToHostApp(
+        content: Any?,
+        onSuccess: (content: Any) -> Unit,
+        onError: (message: String) -> Unit
+    ) {
+        content?.let {
+            val contentJsonStr = it.toString()
+            if (contentJsonStr.isNotBlank()) {
+                onSuccess(contentJsonStr)
+                return
+            }
+        }
+        onError(ErrorBridgeMessage.ERR_UNIVERSAL_BRIDGE)
```

Comment:
I'm not sure if we need to add a default implementation for `sendJsonToHostApp` in SDK side 🤔  if so, then we also need to match the same behaviour with iOS. Or, we can keep this interface unimplemented like `getUniqueId` and let the HostApp to implement.

cc: @rleojoseph @GondiTilak @munir-rakuten

Reply:
There should not be a default implementation for this
```
open fun sendJsonToHostApp(
        info: String,
        onSuccess: (info: String) -> Unit,
        onError: (message: String) -> Unit
    ) {
        throw MiniAppSdkException(ErrorBridgeMessage.NO_IMPL)
    }
```